config/initializers/sentry.rb

```diff
@@ -17,8 +19,16 @@
   # Don't log RetryJobError exceptions in Sentry as they will have already been logged as part of the failed job
   config.excluded_exceptions += ['RetryJobError']
 
-  # Half of all requests will be used in performance sampling.
-  # Currently the MoJ plan does not allow this. Turn on when
-  # plan has been updated in July 2021
```

Comment:
Should we keep this comment?

Reply:
Nah, it was relevant before, but it's now permanently on, so we can get rid of this :)